polars/polars-time/Cargo.toml

```diff
@@ -31,7 +31,7 @@ dtype-duration = ["polars-core/dtype-duration", "polars-core/temporal"]
 rolling_window = ["polars-core/rolling_window", "dtype-duration"]
 private = []
 fmt = ["polars-core/fmt"]
-timezones = ["chrono-tz", "dtype-datetime", "polars-core/timezones"]
+timezones = ["chrono-tz", "dtype-datetime", "polars-core/timezones", "polars-arrow/timezones"]
```

Comment:
Make sure we activate the proper version if we compile with timezones. This was the root of the problem @MarcoGorelli

Reply:
aah nice, thanks Ritchie!